### ISSUE TITLE
Links in Import/Export

### ISSUE THREAD

**Issue by marcusparsons** [AlaSQL/alasql#1496]
The markdown links in the Import/Export section of the Wiki are not links. It's just the title repeated where the link should be.

**Comment by mathiasrw** [comment]
Can you please share a link?

**Comment by marcusparsons** [comment]
Yes, it is in your import and export page for the Wiki. https://github.com/AlaSQL/alasql/wiki/Import-export

<img width="913" alt="Alasql screenshot" src="https://user-images.githubusercontent.com/10608765/192325350-4dfb8ead-c975-4338-b82b-1b3bc4b1930e.png">

**Comment by Parables** [comment]
Can we have a proper documentation site for AlaSQL, perhaps something with [Docusaurus](https://docusaurus.io/) or [Astro](https://astro.build/) @mathiasrw

**Comment by mathiasrw** [comment]
@Parables Absolutely. 

I would suggest a bunch of markdown files that can be feeded into a machine that spits out the final website. Is that something Docusurus or astro offers?

Have a look at https://github.com/AlaSQL/alasql-docs - It never got fully automated in regards to the content, but it does take the wiki and makes it something easier to absorb via http://alasql.surge.sh

**Comment by mathiasrw** [comment]
I asked Agershun to point docs.alasql.com to the online website.

We should move the wiki to the docs repo and have changes be reflected live in the docs.alasql.com

**Comment by mathiasrw** [comment]
It seems to me that we should consider one of these options (in no particular order)

- https://github.com/slatedocs/slate https://slatedocs.github.io/slate/#introduction
- https://docsify.js.org/
- orchid.run https://orchid.run/themes/orchideditorial/demo
- [mkdocs](https://www.mkdocs.org/) (that we are [currently](https://alasql.surge.sh) using)

**Comment by Parables** [comment]
@mathiasrw I looked into the ones you suggested.

Slate is very good for API documentation. 
Looking at the current wiki out of the `382` pages:
About `178` pages are for AlaSQL Keywords
About `125` pages are for SQL Keywords
About `49` pages are for How to Guides

One observation is that most of these keywords are related to each other.
One could potentially have a lot of tabs opened just to digest information on using some of the keywords.

Slate's everything on one page solves this issue. However, written in Ruby and it has issues that make me doubt its reliability
![image](https://user-images.githubusercontent.com/38154990/192725640-6b3e2cfd-5569-48ad-983f-15c95adf4883.png)

I have used docsify [some time ago](https://gun-eco-docs.vercel.app/#/Installation), I have no strong criticism about it just that I feel it's limited.

[Orchid](https://orchid.run/wiki/user-manual/getting-started/quickstart) is Java-based, and that seems like overkill for a JS tool like AlaSQL. The Quick Start guide alone looks quite scary... I wonder where is the quick in that guide

if our objective is to just put some notes out there for people to read, I guess the GitHub wiki does that job perfectly. We only need to order the pages like proper documentation, Include a tutorial section, a FAQ section, an appendix, and some table of contents/sidebar menu

However, you would bear with me that developers go through a lot of stress and poorly organized docs add more salt to their injuries.

I made a little rant on Nhost's docs which you can read [here](https://github.com/nhost/nhost/discussions/905) but well-written, *beautifully presented docs* surely boost developer productivity(DevX) 

Take a look at Astro Build's docs and tell me the impression you had. I choose Astro simply because it is very flexible and framework agnostic. I like tools that impose little or no restrictions/limitations

I selected [Docusaurus](https://docusaurus.io/) because it allows you to quickly build documentation with ease. So if markdown is only what you want Docusaurus is a good fit, with one must-have feature: Documentation versioning.

**Comment by ghost** [comment]
Apologies for not adding anything of substance, but I've just discovered AlaSQL and immediately ran into this. Please consider picking this issue up again for the sake of learners. Thanks!
